resources/instance_image_instance.rb

```diff
@@ -2,8 +2,8 @@
 provides :instance_image_instance
 default_action :create
 
-property :address, String, required: true
-property :subnet, String, required: true
+property :address, String, required: %i(create)
+property :subnet, String, required: %i(create)
```

Comment:
I'd feel better using this notation since we use it globally.
```suggestion
property :address, String, required: [:create]
property :subnet, String, required: [:create]
```